doc/spinel-protocol-src/spinel-prop-mac.md

```diff
@@ -10,15 +10,20 @@ Possible Values:
 * 0: `SCAN_STATE_IDLE`
 * 1: `SCAN_STATE_BEACON`
 * 2: `SCAN_STATE_ENERGY`
+* 3: `SCAN_STATE_DISCOVER`
 
 Set to `SCAN_STATE_BEACON` to start an active scan.
 Beacons will be emitted from `PROP_MAC_SCAN_BEACON`.
 
 Set to `SCAN_STATE_ENERGY` to start an energy scan.
-Channel energy will be reported by alternating emissions
-of `PROP_PHY_CHAN` and `PROP_PHY_RSSI`.
+Channel energy result will be reported by emissions
+of `PROP_MAC_ENERGY_SCAN_RESULT` (per channel).
 
-Values switches to `SCAN_STATE_IDLE` when scan is complete.
+Set to `SCAN_STATE_DISOVER` to start a Thread MLE discovery
+scan operation. Discovery scan result will be emitted from
+`PROP_MAC_SCAN_BEACON`.
```

Comment:
**For the future**: We will need to eventually move all thread-specific parts to the thread-specific document. Perhaps we could define a "technology-specific" scan type or maybe a range of types reserved to be defined by the network technology in use.